### PR TITLE
add post steps for mapping files

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
Fixing the file path issue for `mapping.txt` in circle config

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by Climbatize — APPROVED** [review]
LGTM

File: .circleci/config.yml [hunk 1]
```diff
@@ -36,6 +36,11 @@ workflows:
               only: /^v.*/
             branches:
               only: /.*/
+          post-steps:
+            - persist_to_workspace:
+                root: testapp/build/outputs
+                paths:
+                  - mapping/
```
Comment (corycaywood) [inline .circleci/config.yml:43]: You also need to update the path in the `mapping` jobs: `mapping/staging/mapping.txt`, etc.

Comment (Climbatize) [reply .circleci/config.yml:43]: nice catch @corycaywood , the PR only shows the change so it was not obvious it missed that

**Review by corycaywood — APPROVED** [review]
👍

**Pull request merged**